rs/execution_environment/benches/run-all-benchmarks.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh -eu
```

Comment:
I suppose it's better to use `#!/bin/bash`, like most of the scripts in `ic` repo.